libkbfs/bare_root_metadata.go

```diff
@@ -271,6 +271,12 @@ func (md *BareRootMetadataV2) DeepCopy(codec Codec) (BareRootMetadata, error) {
 	return &newMd, nil
 }
 
+// MakeSuccessor implements the ImmutableBareRootMetadata interface for BareRootMetadataV2.
+func (md *BareRootMetadataV2) MakeSuccessor(codec Codec) (BareRootMetadata, error) {
+	// MDv3 TODO: Make a v3 successor.
+	return md.DeepCopy(codec)
```

Comment:
We have a real MakeSuccessor function around somewhere, right? That increments the version, etc. Why not use that?

Also, don't we still need to continue making V2 successors, at least for a while?